torch/__init__.py

```diff
@@ -38,30 +39,49 @@
 except ImportError:
     pass
 
-# first check if the os package has the required flags
-if not hasattr(_dl_flags, 'RTLD_GLOBAL') or not hasattr(_dl_flags, 'RTLD_LAZY'):
-    try:
-        # next try if DLFCN exists
-        import DLFCN as _dl_flags
-    except ImportError:
-        # as a last attempt, use compile-time constants
-        import torch._dl as _dl_flags
-
-old_flags = sys.getdlopenflags()
-sys.setdlopenflags(_dl_flags.RTLD_GLOBAL | _dl_flags.RTLD_LAZY)
+if platform.system() == 'Windows':
+    # first get nvToolsExt PATH
+    def get_nvToolsExt_path():
+        NVTOOLEXT_HOME = _dl_flags.getenv('NVTOOLSEXT_PATH', 'C:\\Program Files\\NVIDIA Corporation\\NvToolsExt')
```

Comment:
So people on Windows need to always have `NVTOOLSEXT_PATH` set when it's not the default path? Is there any reason why you can't use the helpers from `torch/cuda/nvtx.py`?

Reply:
Yes, it will be okay. This path is quite consistent. Using the helper functions in `torch/cuda/nvtx.py` requires importing `cuda/__init__.py` and that requires importing `torch` first, which is endless loop.